examples/splinterdb_iterators_example.c

```diff
@@ -85,72 +58,47 @@ do_iterate_from(splinterdb *spl_handle, const char *from_key);
 int
 main()
 {
-   printf("     **** SplinterDB Iterators Example program: "
+   printf("     **** SplinterDB Iterators example program: "
           "Lexicographic sort comparison  ****\n\n");
 
-   // Initialize data configuration, describing your key-value properties
+   // Initialize data configuration, using default key-comparison handling.
    data_config splinter_data_cfg;
    default_data_config_init(APP_MAX_KEY_SIZE, &splinter_data_cfg);
 
    // Basic configuration of a SplinterDB instance
    splinterdb_config splinterdb_cfg;
-   configure_splinter_instance(&splinterdb_cfg,
-                               &splinter_data_cfg,
-                               APP_DB_NAME,
-                               (APP_DEVICE_SIZE_MB * K_MiB),
-                               (APP_CACHE_SIZE_MB * K_MiB));
+   memset(&splinterdb_cfg, 0, sizeof(splinterdb_cfg));
```

Comment:
Removed wrapper `configuration` method and inline'd code to shorten the whole program.